contrib/upstart-recursor.conf

```diff
@@ -1,9 +1,9 @@
 # recursor - the PowerDNS Recursor
 # for in /etc/init/recursor.conf
 description "PowerDNS Recursor"
+start on static-network-up
```

Comment:
According to [this](http://upstart.ubuntu.com/cookbook/#standard-idioms) you could use `start on stopped network` in older versions. As the static-network-up refers to the interfaces listed in `/etc/network/interfaces`.